buildSrc/src/main/java/baritone/gradle/task/ProguardTask.java

```diff
@@ -163,15 +170,13 @@ private void acquireDependencies() throws Exception {
 
         // Iterate the required libraries to copy them to tempLibraries
         for (String lib : this.requiredLibraries) {
-            // Download the version jar from the URL acquired from the version manifest
-            if (lib.startsWith("minecraft")) {
-                String version = lib.split("-")[1];
-                Path versionJar = getTemporaryFile("tempLibraries/" + lib + ".jar");
-                if (!Files.exists(versionJar)) {
-                    JsonObject versionJson = PARSER.parse(new InputStreamReader(new URL(this.versionDownloadMap.get(version)).openStream())).getAsJsonObject();
```

Comment:
if version download map is no longer used, it should probably be removed from the task, fields, and build.gradle stuff